### PR TITLE
ClientSideJS

### PR DESCRIPTION
**Pull request by tpryan**
This is a quick and dirty implementation of client side JS use of Cloud Vision API

### CODE REVIEW

**Comment by tpryan** [comment]
I'm a google employee. Already covered by CLA.

File: js/README.MD [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+This directory contains [Cloud Vision API](https://cloud.google.com/vision/) Client Side JS samples.
+
+## Prereqisites
+1. [Sign up for Google Cloud Platform Account](http://cloud.google.com)
+2. [Enable Cloud Vision API](https://cloud.google.com/vision/docs/getting-started)
+
+
+
+## Setup
+1. Create a key.js containing your API key.
```
Comment (tswast) [inline js/README.MD]: Maybe add more instructions like you have in the comment of `main.js`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+$(document).ready(function() {
+  $('form').on('submit', uploadFiles);
+});
+
+function uploadFiles(event) {
+  event.stopPropagation(); // Stop stuff happening
+  event.preventDefault(); // Totally stop stuff happening
+
+  //Grab the file and asynchronously convert to base64.
```
Comment (tswast) [inline js/main.js]: nit. Be consistent with spaces after //, so // Grab

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    '	}' +
+    ']' +
+    '}';
+
+  $.ajax({
+    type: 'POST',
+    url: cvurl,
+    dataType: 'json',
+    data: json,
+    //Include headers, otherwise you get an odd 400 error.
```
Comment (tswast) [inline js/main.js]: nit. // Include

**Comment by tpryan** [comment]
@tswast I made the changes you requested.  Let me know if I went far enough.

**Comment by tswast** [comment]
Looks Good To Me. Thanks!

**Pull request merged**